<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph2DWindow</class>
 <widget class="QWidget" name="Graph2DWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QChartView" name="graphics"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="xmin">
       <item>
        <widget class="QPushButton" name="xmin_inf">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="xmin_text">
         <property name="text">
          <string>xMin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="xmin_sup">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="xmax">
       <item>
        <widget class="QPushButton" name="xmax_inf">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="xmax_text">
         <property name="text">
          <string>xMax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="xmax_sup">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ymin">
       <item>
        <widget class="QPushButton" name="ymin_inf">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ymin_sup">
         <property name="text">
          <string>yMin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="xmin_sup_2">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="ymax">
       <item>
        <widget class="QPushButton" name="ymax_inf">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ymax_text">
         <property name="text">
          <string>yMax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ymax_sup">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
